app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from random import choice
+import matplotlib.pyplot as plt
+
+
+def flip_coin():
+    head_percentage = dict()
+    heads = 0
+    for time in range(10000):
+        heads = len([heads for _ in range(10) if choice((0, 1)) == 0])
+        head_percentage[heads] = head_percentage.get(heads, 0) + heads
```

Comment:
Whe do you add `heads` to `head_percentage[heads]`? Percentage of 0 heads flips is always 0% in your code.